CHANGELOG.md

```diff
@@ -6,6 +6,31 @@ Embedded Graphics is a `no_std` library for adding graphics features to display
 
 ## [Unreleased] - ReleaseDate
 
+### Added
+- Made `Size::from_bounding_box` public
```

Comment:
The CHANGELOG should reflect API changes and from a user perspective it doesn't matter that this function has been available internally before:

```suggestion
- [#794](https://github.com/embedded-graphics/embedded-graphics/pull/794) Added `from_bounding_box` constructor to `Size`.
```